tests/unit/modules/network/sonic/fixtures/sonic_aaa.yaml

```
@@ -0,0 +1,79 @@
+---
+merged_01:
+  module_args:
+    config:
+      authentication: 
+        data:
+          local: true
```

local and group attributes have been consolidated into one attribute ```suggestion
         default_auth:
             - local
             - radius
```

---

@stalabi1 : Modified the UT cases based on the recent changes in aaa module.